reactive/reactive.js

```diff
@@ -120,6 +120,7 @@ module('users.timfelgentreff.reactive.reactive').requires('users.timfelgentreff.
 	        var cobj = new Constraint(func, this);
 			cobj.allowFailing = true;
 	        cobj.addPrimitiveConstraint(new ReactiveSolver.Constraint(this, cobj, func));
+	        cobj.opts = opts;
```

Comment:
with the latest master merge, _options was introduced... Considering the thing is called opts from the beginning, I actually prefer your naming, but we should find a consistent solution